src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp

```diff
@@ -1334,10 +1334,18 @@ void BrgemmCopyBEmitter::execute(matmul::jit_brgemm_matmul_copy_b_t *kernel, con
     (*kernel)(&ctx);
 }
 
-HorizonMaxEmitter::HorizonMaxEmitter(dnnl::impl::cpu::x64::jit_generator* h, dnnl::impl::cpu::x64::cpu_isa_t isa, const std::shared_ptr<ov::Node>& n) :
-    jit_emitter(h, isa, n, Precision::FP32, emitter_in_out_map::vec_to_vec) {}
+HorizonEmitter::HorizonEmitter(dnnl::impl::cpu::x64::jit_generator* h, dnnl::impl::cpu::x64::cpu_isa_t isa, const std::shared_ptr<ov::Node>& n) :
+    jit_emitter(h, isa, n, Precision::FP32, emitter_in_out_map::vec_to_vec) {
+    if (ov::is_type<const snippets::op::HorizonMax>(n)) {
+        m_op_type = OpType::max;
+    } else if (ov::is_type<const snippets::op::HorizonSum>(n)) {
+        m_op_type = OpType::sum;
+    } else {
+        OPENVINO_THROW("HorizonEmitter exprects HorizonMax or HorizonSum ops");
```

Comment:
`exprects` -> `supports`?

Reply:
As it's already been merged, I'll cover this typo inside other snippets PR. Thanks Dmitry!